recipe/meta.yaml

```diff
@@ -39,7 +39,8 @@ source:
   # https://github.com/numpy/numpy/tree/v{{ version }}/vendored-meson
   - folder: vendored-meson/meson
     git_url: https://github.com/numpy/meson.git
-    git_rev: 6f88e485f27bb0a41d31638f0c55055362e0b1ac
+    git_rev: 6f88e485f27bb0a41d31638f0c55055362e0b1ac  # [not (win and is_freethreading)]
+    git_rev: 5f7159556a40844cba411e3a45d61192e36bf9ad  # [win and is_freethreading]
```

Comment:
Where did you get that commit from? I don't find it in https://github.com/numpy/meson

Reply:
To find commits you need to paste them after the `/commit/` part of the URL. Here it is:

https://github.com/numpy/meson/commit/5f7159556a40844cba411e3a45d61192e36bf9ad

Reply:
Thanks. I do know that part, but what I meant by "I don't find it" is the yellow box here

![image](https://github.com/user-attachments/assets/71bd3093-754d-4381-9098-08927652e21e)

It's not an obvious branch/tag, nor does the submodule in numpy point to that commit, so I'm curious where this comes from. Probably some PR somewhere?

Reply:
Came from https://github.com/numpy/meson/pull/15

Reply:
The message is just that. The commit is not on that repository yet. As this is all experimental and new I would expecte it to be in a PR, as Isuru just posted above.

Reply:
> As this is all experimental and new I would expecte it to be in a PR, as Isuru just posted above.

Which is what I was interested in. ;)

I'd leave a comment, but not a blocking concern.